matter/si91x/siwx917/BRD4338A/autogen/sl_component_catalog.h

```diff
@@ -3,26 +3,28 @@
 
 // APIs present in project
 #define SL_CATALOG_BTN0_PRESENT
-#define SL_CATALOG_BUTTON_PRESENT
-#define SL_CATALOG_BUTTON_BTN0_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_BTN0_PRESENT
 #define SL_CATALOG_BTN1_PRESENT
-#define SL_CATALOG_BUTTON_BTN1_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_BTN1_PRESENT
 #define SL_CATALOG_LED0_PRESENT
-#define SL_CATALOG_LED_PRESENT
-#define SL_CATALOG_LED_LED0_PRESENT
+#define SL_CATALOG_SIMPLE_LED_PRESENT
+#define SL_CATALOG_SIMPLE_LED_LED0_PRESENT
 #define SL_CATALOG_LED1_PRESENT
-#define SL_CATALOG_LED_LED1_PRESENT
-#define SL_CATALOG_TIMER0_PRESENT
-#define SL_CATALOG_ULP_TIMERS_INSTANCE_PRESENT
-#define SL_CATALOG_TEMP_ULP_TIMER_TIMER0_PRESENT
+#define SL_CATALOG_SIMPLE_LED_LED1_PRESENT
+#define SL_CATALOG_ULP_TIMER_PRESENT
+#define SL_CATALOG_ULP_TIMER_TIMER0_PRESENT
 #define SL_CATALOG_CMSIS_OS_COMMON_PRESENT
 #define SL_CATALOG_DEVICE_INIT_NVIC_PRESENT
 #define SL_CATALOG_EMLIB_CORE_PRESENT
 #define SL_CATALOG_EMLIB_CORE_DEBUG_CONFIG_PRESENT
 #define SL_CATALOG_FREERTOS_KERNEL_PRESENT
 #define SL_CATALOG_KERNEL_PRESENT
+#define SL_CATALOG_GLIB_PRESENT

```

Comment:
Do we know if this is necessary if the LCD is disabled?

Reply:
probably not

Reply:
actually this one is duplicated. there was one in the #ifdef DISPLAY_ENABLED that i didn't spot. Removed it